src/components/Authentication/Login.jsx

```diff
@@ -12,11 +12,15 @@ import {
   Button,
   Flex,
   Link,
+  Text,
   Center,
+  Container,
+  Stack,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Stack' is defined but never used.